### PR TITLE
Docs: Add new SQS sample (listen from multiple queues)

### PR DESCRIPTION
**Pull request by mcruzdev**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring

## :scroll: Description

This pull request adds a sample for SQS where is shows a listener receiving messages from multiple queues.

## :bulb: Motivation and Context

The motivation is described in issue #670.

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated reference documentation to reflect the change
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import software.amazon.awssdk.services.sqs.model.Message;
+
+import java.util.UUID;
+
+@SpringBootApplication
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java:30]: Having multiple main classes breaks the build. Can we have this sample working as a part of the other main class?

**Review by maciejwalkowiak — CHANGES_REQUESTED** [review]

File: spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java (hunk 1 above)
Comment (mcruzdev) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java]: Hi @maciejwalkowiak, thank you for you review.

I think that is not necessary to add `@Order` for ApplicationRunner bean.

> Interface used to indicate that a bean should run when it is contained within a [SpringApplication](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/SpringApplication.html). Multiple [ApplicationRunner](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/ApplicationRunner.html) beans can be defined within the same application context and can be ordered using the [Ordered](https://docs.spring.io/spring-framework/docs/6.0.7/javadoc-api/org/springframework/core/Ordered.html) interface or [@Order](https://docs.spring.io/spring-framework/docs/6.0.7/javadoc-api/org/springframework/core/annotation/Order.html) annotation.

What do you think?

Comment (maciejwalkowiak) [reply spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSqsListenMultipleQueues.java]: Only if its relevant for the sample. If sample can work without `@Order` then its better without.

Once you removed `@SpringBootApplication` i think this class is not picked at all right? It must have `@Configuration`

**Comment by tomazfernandes** [comment]
@mcruzdev sorry for the delay and thanks for the PR!

Looks good to me, if you can just add yourself as the author.

@maciejwalkowiak other than the author tag I've nothing else to add, feel free to merge when you're ok with it.

**Pull request merged**